packages/design-system-react/src/components/avatar-base/AvatarBase.stories.tsx

```diff
@@ -81,27 +81,27 @@ export const Size: Story = {
         <AvatarBase
           shape={AvatarBaseShape.Square}
           size={AvatarBaseSize.Xs}
-          fallbackText="XS"
+          fallbackText="Xs"
```

Comment:
Showing that uppercase class works as expected

<img width="625" alt="Screenshot 2025-02-04 at 3 57 03 PM" src="https://github.com/user-attachments/assets/7e96e053-c00c-4a27-b005-5cd66602886f" />

Reply:
this looks so gross @.@